roles/edpm_update/tasks/packages.yml

```diff
@@ -1,5 +1,16 @@
 ---
 
+- name: Import edpm_ovs role for independent openvswitch update
+  ansible.builtin.import_role:
+    name: osp.edpm.edpm_ovs
+    tasks_from: update
+
+- name: Ensure openvswitch is excluded from bulk update
```

Comment:
The package was just updated by the previous task that imported `ovs` role. Why do we need to exclude the package here?

Reply:
Was this addressed?

Reply:
I had comments in pending. See my response from 2 days ago

Reply:
Actually, i see what you are saying now. The previous task should have already updated OVS so even if the `Apply packages updates` runs with OVS it will be no-op since it has been already updated. I still prefer to have this extra guard but I can drop it if you want